### DISCUSSION TITLE
Portable version

### DISCUSSION THREAD

**Discussion by lmroces** [audiamus/AaxAudioConverter#163] [Ideas]
Firstly, thanks for this great program! I love it as it enables me to convert my audiobooks in a format compatible with  my mp3 player. When I install the program, however, it doesn't ask where to install it. Would you consider adding this possibility, or making a portable version? Again, thanks!

**Comment by audiamus** [comment]
I'm afraid that won't happen. Both AAX Audio Converter and Book Lib Connect are very much built on top of the Windows eco system, for simplicity and efficiency. The GUI in particular uses Windows Forms which makes it platform-dependent. A new implementation with a XAML-based framework would be a lot of effort.

**Comment by lmroces** [comment]
Thank you for your kind and quick answer! And thanks for this program!
I hope you had a merry Christmas day!

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------

**Comment by Henry-Jones** [comment]
I use it for quite a while as a portable (in the sense "on different Windows PC's") app. As a lot of my portable apps use ffmpeg as well i set the correct path on a mobile drive and then copy the content of the settings file to the one in the exe folder.

Changed settings are still written to the machine I'm on, but this does not concern me.
